siliconcompiler/core.py

```diff
@@ -2987,15 +3004,14 @@ def clock(self, pin, period, jitter=0):
         self.set('datasheet', design, 'pin', pin, 'tjitter', 'global', jitter_range)
 
     ###########################################################################
-    def node(self, flow, step, tool, index=0):
+    def node(self, flow, step, tool, task=None, index=0):
```

Comment:
Does it make sense to have task==None, it seems like this is a required parameter

Reply:
Sounds good, do you think you can fix the tests/flows that call (). They will break if we make task required.